CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [v0.8.2] - 2020-06-17
+
+- Fix template helpers issue converting port number for the webhook endpoint.
```

Comment:
```suggestion
### Fixed

- Fix template helpers issue converting port number for the webhook endpoint.
```

Last two releases and this one being prepared introduced slight inconsistency with the format/structure of the change log - we should use these listing headings as in https://keepachangelog.com/en/1.0.0/

We can fix the other two in a follow-up PR.